docs/nodes/AI_ML/CLASSIFICATION/SUPPORT_VECTOR_MACHINE/examples/EX1/example.md

```diff
@@ -1,3 +1,3 @@
-In this example, the `SUPPORT_VECTOR_MACHINE` is passed the [iris dataset](https://archive.ics.uci.edu/dataset/53/iris), split into two parts. The training data contains 120 labels examples, while the input dataset contains 30 samples with the labels removed.
+In this example, the `SUPPORT_VECTOR_MACHINE` is passed the [iris dataset](https://archive.ics.uci.edu/dataset/53/iris), split into two parts. The training data contains examples of 120 labels, while the input dataset contains 30 samples with the labels removed.
```

Comment:
Are they "examples of labels" or just "labels"? My feeling is that they are actually labels, in which case we should remove "examples of".